src/test/system_tests.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+//
+#include <test/util/setup_common.h>
+#include <util/system.h>
+#include <univalue.h>
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(system_tests, BasicTestingSetup)
+
+// At least one test is required (in case HAVE_BOOST_PROCESS is not defined).
+// Workaround for https://github.com/bitcoin/bitcoin/issues/19128
+BOOST_AUTO_TEST_CASE(dummy)
+{
+    BOOST_CHECK(true);
+}
+
+#ifdef HAVE_BOOST_PROCESS
+BOOST_AUTO_TEST_CASE(run_command)
+{
+    {
+        const UniValue result = RunCommandParseJSON("");
+        BOOST_CHECK(result.isNull());
+    }
+    {
+#ifdef WIN32
+        // Windows requires single quotes to prevent escaping double quotes from the JSON...
+        const UniValue result = RunCommandParseJSON("echo '{\"success\": true}'");
+#else
+        // ... but Linux and macOS echo a single quote if it's used
+        const UniValue result = RunCommandParseJSON("echo \"{\"success\": true}\"");
+#endif
+        BOOST_CHECK(result.isObject());
+        const UniValue& success = find_value(result, "success");
+        BOOST_CHECK(!success.isNull());
+        BOOST_CHECK_EQUAL(success.getBool(), true);
+    }
+    {
+        BOOST_REQUIRE_THROW(RunCommandParseJSON("invalid_command"), std::runtime_error); // Command failed
```

Comment:
In commit `[util] add RunCommandParseJSON`

Maybe use the `false` command which always fails? Or is the intent to try a command that probably won't exist?

Reply:
Keeping it for now, but I suppose `false` could work too, assuming it exists on Windows.